test/SourceKit/InterfaceGen/error_swift_module.swift

```diff
@@ -45,14 +45,14 @@ import NonExistant
 // RUN: not %sourcekitd-test -req=interface-gen -module LibraryWrong -- -I %t -target %target-triple %s 2>&1 | %FileCheck --check-prefix TRANSITIVE-FAIL %s
 // TRANSITIVE-FAIL: Could not load module: LibraryWrong (cannot load module 'Transitive' as 'WrongName')
 
-// Try a non-existant module
+// Try a non-existent module
 
-// RUN: not %sourcekitd-test -req=interface-gen -module NonExistant -- -I %t -target %target-triple %s 2>&1 | %FileCheck --check-prefix DIRECT-NONEXISTANT %s
-// DIRECT-NONEXISTANT: Could not load module: NonExistant
+// RUN: not %sourcekitd-test -req=interface-gen -module NonExistent -- -I %t -target %target-triple %s 2>&1 | %FileCheck --check-prefix DIRECT-NONEXISTENT %s
+// DIRECT-NONEXISTENT: Could not load module: NonExistent
 
-// RUN: %target-swift-frontend -emit-module %t/Inputs/Transitive.swift -module-name NonExistant -o %t
-// RUN: %target-swift-frontend -emit-module %t/Inputs/LibraryNonExistant.swift -module-name LibraryNonExistant -I %t -o %t
+// RUN: %target-swift-frontend -emit-module %t/Inputs/Transitive.swift -module-name NonExistent -o %t
+// RUN: %target-swift-frontend -emit-module %t/Inputs/LibraryNonExistant.swift -module-name LibraryNonExistent -I %t -o %t
 // RUN: rm -rf %t/NonExistant.swiftmodule
```

Comment:
You missed two renames here. While debugging the failure, I also noticed that we were carrying state between executions of this test. Could you fix that as well by adding `// RUN: %empty-directory(%t)` as the first line of this test file (before `// RUN: %empty-directory(%t/Inputs)`).

```suggestion
// RUN: %target-swift-frontend -emit-module %t/Inputs/LibraryNonExistent.swift -module-name LibraryNonExistent -I %t -o %t
// RUN: rm -rf %t/NonExistent.swiftmodule
```